app/src/main/java/com/appunite/loudius/ui/authenticating/AuthenticatingViewModel.kt

```diff
@@ -53,14 +62,16 @@ sealed class AuthenticatingScreenNavigation {
 
 class AuthenticatingViewModel(
     private val authRepository: AuthRepository,
-    savedStateHandle: SavedStateHandle
+    savedStateHandle: SavedStateHandle,
+    private val eventTracker: EventTracker
 ) : ViewModel() {
 
     private val code = Screen.Authenticating.getCode(savedStateHandle)
     var state by mutableStateOf(AuthenticatingState())
         private set
 
     init {
+        eventTracker.trackEvent(AuthenticationStartedEvent)
```

Comment:
**suggestion**: You can consider moving this event into `getAccessToken()` where actually the refresh token is being exchanged